kolibri/core/assets/src/views/SideNav.vue

```diff
@@ -1,6 +1,12 @@
 <template>
 
-  <div ref="sideNav" class="side-nav-wrapper" tabindex="0" @keyup.esc="toggleNav">
+  <div
+    v-show="navShown"
```

Comment:
What does this `v-show` do?

Reply:
In a previous issue, I added the tabindex in order to close the side menu with the escape button. Because of that, you have to press the tab key 2 times in order to go from the navbar to the main content. So with the navShow the div can be hidden until you open the menu.